### ISSUE TITLE
Extension needs an update, after recent WhatsApp Web update

### ISSUE THREAD

**Issue by krashish8** [tomer8007/whatsapp-web-incognito#40]
Error in the JavaScript console:
```
interception.js:916 WhatsAppWebIncognito: Can't find the WhatsApp API. Sending read receipts might not work.
```
![image](https://user-images.githubusercontent.com/39548570/109101851-8efc0400-774d-11eb-8d68-23e3d1578b94.png)

I guess the class names in the `core/ui_class_names.js` file shall be updated, because after running `document.getElementsByClassName("className")`, I can't get any element in the JavaScript console.

**Comment by krashish8** [comment]
Also, how do I get the WhatsApp web version? I am using the browser version of WhatsApp Web: https://web.whatsapp.com/

**Comment by rohanchkrabrty** [comment]
@krashish8 both are same.

**Comment by dgadelha** [comment]
> 
> 
> Also, how do I get the version number of the WhatsApp web which I am using: [web.whatsapp.com](https://web.whatsapp.com/)

Go to Settings -> Help
![image](https://user-images.githubusercontent.com/10081640/109149249-0fb70200-7746-11eb-9cb0-352ea8bfa490.png)

**Comment by krashish8** [comment]
Thanks @dgadelha That's the version I'm using.
By the way, is the version decided based on the version of the android app used? So, can I revert back to previous version of whatsapp web on my "browser"?

**Comment by elbeshenrique** [comment]
I'm with the same problem here

**Comment by tomer8007** [comment]
Yes, this is now fixed in 6eb2c4efd0339b8dd667a03999cae0c9bc30b339. I need to make the UI class names automatically identified in the future.

**Comment by krashish8** [comment]
Thanks for the fix. Well, that's a good idea, it would be better if the UI class names are identified automatically.
